sdk/compiler/damlc/tests/daml-test-files/CCTPMintToken.daml

```diff
@@ -58,6 +58,8 @@ main =
       invalidSignature = "30450221009944dfb75e1e46006dd6922ba41145ab47dd00da2af75027a95589764fb5e72902200ac5c17d31d102dbdde75e4f5809dccbba9a118cf4a6bffd50108e8444bb69b5"
   in
     script do
+      -- Secp256k1 signing is non-deterministic, so we simply validate that our daml-script question runs successively here
```

Comment:
Secp256k1 is non-deterministic (as random data is typically mixed into the signing flow). So we simply validate that this question runs here.

Reply:
Does the crypto implementation allow you to specify the source of randomness? You could fix that one's seed to make it deterministic for testing here; and for other customers.

Reply:
Thanks for that suggestion. After some digging around, managed to figure out how to make `SecureRandom` behave deterministically!